### ISSUE TITLE
🐛 Bug Report: Appwrite Cloud Python 3.9 function module not found in tarball

### ISSUE THREAD

**Issue by tripolskypetr** [appwrite/appwrite#6051]
### 👟 Reproduction steps

I am trying to import `repository.py` from `index.py`

```python
tripolskypetr@desktop:/mnt/c/appwrite/functions/paginator$ ls
__pycache__  index.py  main.py  repository.py  requirements.txt
tripolskypetr@desktop:/mnt/c/appwrite/functions/paginator$ cat index.py
import sys

from appwrite.exception import AppwriteException

from repository import get_repository_instance

def main(req, res):

    repository = get_repository_instance()

    try:
        repository.try_init(req)
        repository.read_event(req)
    except AppwriteException as err:
        print(err.message)

    print(repository.documents)

    return res.json({
        "areDevelopersAwesome": True,
    })
tripolskypetr@desktop:/mnt/c/Users/tripolskypetr/Documents/GitHub/appwrite/functions/paginator$ 

```

It works on a local mock environment

```python
tripolskypetr@desktop:/mnt/c/appwrite/functions/paginator$ cat main.py
import mock

from dotenv import dotenv_values

from index import main

req = mock.Mock()
req.headers = {}
req.payload = ""
req.variables = dict(dotenv_values(".env"))

res = mock.Mock()
res.send = lambda text, status = "" : print(text, status)
res.json = lambda data, status = "" : print(data, status)

main(req, res)
```

But when I deploy that function to appwrite cloud It fails with `ModuleNotFoundError`

```text
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'repository'
```

### 👍 Expected behavior

Looks like python function container bootstrap contains these files

```python
print(os.listdir())
['userlib', '.env', '.dockerignore', 'docker-compose.yml', 'server.py', 'start.sh', 'requirements.txt', 'README.md', '.gitignore', 'build.sh', 'Dockerfile']
```

If I `listdir` the `userlib` folder I will get my actual tarball containment

```python
print(os.listdir("./userlib"))
['requirements.txt', 'main.py', '__pycache__', 'repository.py', 'runtime-env', 'index.py']
```

Using `sys.path.append("userlib")` fixed the import problem. That should be done by default in `server.py`

```python
import sys

sys.path.append('userlib')

from appwrite.exception import AppwriteException

from repository import get_repository_instance

"""
Works well
"""

```

### 👎 Actual Behavior

1. Please add `sys.path.append("userlib")` in `server.py` to allow code spit between multiple files

2. Please fix empty logs for module import error in python functions

![image](https://github.com/appwrite/appwrite/assets/19227776/9dbb1b5e-e790-4962-95a7-cfda4f37ba1a)

### 🎲 Appwrite version

Appwrite Cloud

### 💻 Operating system

Linux

### 🧱 Your Environment

cloud.appwrite.io version 1.1.2

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by tripolskypetr** [comment]
I think this is related to [this issue](https://github.com/appwrite/appwrite/issues/6051)

**Comment by joeyouss** [comment]
Hi @tripolskypetr , just to be sure, for requests, have you used this? https://pypi.org/project/requests/

**Comment by joeyouss** [comment]
Also, @tripolskypetr , about the second thing related to error logs, yes, we are aware of it and this should be fixed in 1.4. Adding PR here: https://github.com/appwrite/appwrite/pull/5725

**Comment by joeyouss** [comment]
Ok cool, I just confirmed and that is also fixed in 1.4, here is a relevant PR: https://github.com/open-runtimes/open-runtimes/pull/137

**Comment by joeyouss** [comment]
Closing this issue following the above comments.
